### DISCUSSION TITLE
Welcome to Minecraft-Server-Launcher Discussions!

### DISCUSSION THREAD

**Discussion by Waheal** [MSLTeam/MSL#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xiaochen296** [comment]
首先我发现一个问题，有些Frp官方ini格式的配置文件导入后无法启动，但是手动输入相应的配置可以启动内网穿透，我认为是MSL开服器的问题，可以优化一下配置文件的识别
